docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+physical topology.”*
+
+![OU Stands for ‘Organizational Unit’](../media/Untitled.png)
+
+
+[From MS Docs](https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc780036(v=ws.10)#structure-and-storage-technologies)
+
+*“**AD DS** uses **domains** and **forests** to represent the **logical structure** of the directory hierarchy.
+**Domains** are used to manage the various populations of users, computers, and network resources in your enterprise.
+The **forest represents the security boundary for AD DS**. Within Domains, you can create **organizational units** to
+subdivide the various divisions of administration.*
+
+*The **logical structure** of AD DS includes a two-dimensional definition that can be viewed as a hierarchy, even though
+the objects themselves are stored in a flat database file. In addition to its own name, **each object stores the name of
+the container directly above it in the hierarchy. That container object stores the name of its superior container,
+and so on, up to the root container. In this way, a logical structure is imposed that can be viewed by using AD DS tools
+as a tree of containers**. By virtue of a hierarchical naming system, the objects in the tree appear to be nested inside
+(contained by) other objects.”*
+
+The main components of the Active Directory logical structure are described as follows
+
+- Active Directory Forest
+    - The highest level in the hierarchy. A forest represents a collection of Directories. It is a security boundary, 
+    which gives administrators complete control and access to all the information inside both the Forest and all
+    Domains within the Forest. Multiple Forests may exist at a time.
+- Domain
+    - Domains partition the information that is stored inside the directory into smaller portions so that the
+  information can be more easily stored on various domain controllers and so that administrators have a greater
+  degree of control over replication. Data that is stored in the Active Directory is replicated throughout the forest from
+  one domain controller to another. Some data that is relevant to the entire forest is replicated to all domain
+  controllers. Other data that is relevant only to a specific domain is replicated only to domain controllers in
+  that particular domain. A good domain design makes it possible to implement an efficient replication topology.
+  This is important because it enables administrators to manage the flow of data across the network, that is, to
+  control how much data is replicated and where that replication traffic takes place.
+- Organizational Unit (OU)
+    - OU’s provide a way to group resources so that they can be managed as a single unit. This allows admins to apply
+group policies across many computers or control access of resources for many users. OUs can also be delegated to 
+specific admins, so that they only have control over their subset of resources.
```

Comment:
I'm not quite sure from this description what OUs are. Are they a concept that exists on a forest / domain level or can you group together any set of resources in an AD across forests or domains?

Would be great if this can be part of the analogy recommended above.